webpack/assets/javascripts/react_app/components/AutoComplete/__tests__/__snapshots__/AutoCompleteReducer.test.js.snap

```diff
@@ -52,25 +50,18 @@ Object {
 
 exports[`AutoComplete reducer should handle AUTO_COMPLETE_SUCCESS 1`] = `
 Object {
-  "controller": null,
+  "controller": undefined,
```

Comment:
I think you have a bug - on success I would imagine `controller` be a string and neither `null` nor `undefined`.